README.md

```diff
@@ -1,65 +1,643 @@
+
+The values are (hopefully) mostly self-explaining. Those you may want to experiment with changing are:
+
+- `PRETALX_VERSION`, the Pretalx version to install into the image
+- `BASE_IMAGE`, the base image to build from
+- `BASE_TAG`, the base image tag to build from
+- `PRETALX_BASE_IMAGE`, the name of the pretalx base image to build
+- `PRETALX_BASE_TAG`, the tag of the pretalx base image to build
+- `PRETALX_IMAGE`, the name of the pretalx image to build
+- `PRETALX_TAG`, the tag of the pretalx image to build
+
+### Local building of the Container images with using the Compose manifests
+
+Use the `build*` scripts in the `bin/` repository to conduct local builds, which package the most common use cases.
+
+```sh
+bin/build
+bin/build.standalone
+bin/build.source
+```
+
+The commands below help to develop and debug the Compose manifests and are provided as a reference:
+
+```sh
+docker compose --env-file .env -f compose.yml --env-file .env.build -f compose/build/default.yml build app
+```
+
+This will build the image with the name `${PRETALX_IMAGE}:${PRETALX_TAG}`, as specified by the inferred `image:` directive.
+
+### Building different combinations of overlays
+
+There is a need to accommodate for the presence of Pretalx plugins together with the application.
+
+This is achieved by creating overlay OCI file system layers and building a custom container image based on the default build context.
+
+```sh
+docker compose --env-file .env -f compose.yml -f compose/local.yml --env-file .env.build -f compose/build/extended.yml config
+```
+
+Or in a live environment:
+
+```sh
+docker compose --env-file .env -f compose.yml -f compose/traefik.yml --env-file .env.build -f compose/build/extended.yml config
+```
+
+All Compose commands in place of `config` apply from here.
+
+If you had successfully built your local Pretalx image, you could build an image extended with plugins with:
+
+```sh
+docker compose -f compose.yml --env-file .env.build -f compose/build/extended.yml build app
+```
+
+## Running
+
+During development, you may want to watch your container scheduler for the health of the containers from another shell:
+
+```sh
+watch -n 0.5 docker ps
+```
+
+### Locally
+
+When you are done with building and preloading the images into your container engine's image store, you can start the composition with:
+
+```sh
+docker compose --env-file .env -f compose.yml -f compose/local.yml --env-file .env.build -f compose/build/extended.yml up -d
+```
+
+> - *Continue* to **Initialisation** below.
+
+---
+
+### Local live-like environment
+
+If you were running a local `traefik` instance on a local `web` network, maybe even with a Smallstep CA for provisioning ACME certificates for your `.internal` network, you could add the network and necessary labels with:
+
+```sh
+docker compose -f compose.yml -f compose/local.yml -f compose/traefik.yml config
+```
+
+### Live
```

Comment:
Good spot! You catched me here being weary of that term. I'm choosing live, because for me running a web application is more like publishing a live radio or television show, which needs constant attention, rather than delivering a fixed asset, such as a product. Since we're at it, I'm also weary of concealing the fact of human labour involved by using corporate language, which often dehumanises processes.

If others insist on this, I'm willing to change the phrasing to the more commonplace industry convention.